OrionUO/Wisp/WispWindow.cpp

```diff
@@ -333,13 +333,11 @@ LRESULT CWindow::OnWindowProc(HWND &hWnd, UINT &message, WPARAM &wParam, LPARAM
         {
             if (LOWORD(lParam) == HTCLIENT)
             {
-                ::ShowCursor(FALSE);
-                return 0;
-            }
-
-            ::ShowCursor(TRUE);
-            break;
-        }
+				SetCursor(NULL);
+				return 0;
+			}
+			break;
+		}
```

Comment:
Can you fix the indentation to use spaces instead? The travis-ci formatting check is failing. 

I recommend you to use clang-format plugin on Visual Studio: https://marketplace.visualstudio.com/items?itemName=LLVMExtensions.ClangFormat

Does this works better? On my Windows 10 I had huge delays when windows enabled/disabled default cursor this way.